source/upgrade.txt

```diff
@@ -104,6 +104,16 @@ Version 5.0 Breaking Changes
      
      VectorSearchOptions opts = vectorSearchOptions().filter(eq("x", 8));
 
+- This driver removes the
+  ``org.mongodb.scala.ObservableImplicits.ToSingleObservableVoid`` implicit
+  class, which means that ``org.reactivestreams.Publisher[Void]`` is no longer
+  automatically convertible to ``org.mongodb.scala.SingleObservable[Void]``. The
+  API exposes ``org.mongodb.scala.Observable[Unit]`` instead of
+  ``org.mongodb.scala.Observable[Void]``. 
+  
+  .. After the 5.0 Scala API docs are released, this line should be added. 
+    For more information, see the `Observable trait documentation <https://mongodb.github.io/mongo-java-driver/5.0/apidocs/mongo-scala-driver/org/mongodb/scala/Observable.html>`__.
```

Comment:
This link doesn't currently work because there's no version 5.0
You can see the link here: https://mongodb.github.io/mongo-java-driver/4.11/apidocs/mongo-scala-driver/org/mongodb/scala/Observable.html

It should be uncommented once the Scala 5.0 documents have been released, per https://jira.mongodb.org/browse/DOCSP-35704